x/evm/keeper/utils_test.go

```diff
@@ -207,7 +207,8 @@ func (suite *KeeperTestSuite) TestCheckSenderBalance() {
 	vmdb.AddBalance(suite.address, hundredInt.BigInt())
 	balance := vmdb.GetBalance(suite.address)
 	suite.Require().Equal(balance, hundredInt.BigInt())
-	vmdb.Commit()
+	err := vmdb.Commit()
```

Comment:
This addressed a warning that there was an unchecked error. This should not raise an error but added the check for safety.